integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOCILoadBalancer.java

```diff
@@ -0,0 +1,321 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.dockerLoginAndPushImageToRegistry;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOCILoadBalancer;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.setPodAntiAffinity;
+import static oracle.weblogic.kubernetes.utils.TestUtils.callWebAppAndCheckForServerNameInResponse;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+
+/**
+ * Verify OCI Load Balancer is installed and running.
+ * Verify sample-war web application be accessed via OCI LoadBalancer.
+ * Verify Load Balancing between two managed servers in the cluster
+ */
+@DisplayName("Verify the sample-app app can be accessed from "
+    + "all managed servers in the domain through OCI Load Balancer")
+@IntegrationTest
+class ItOCILoadBalancer {
+
+
+  // domain constants
+  private static final int replicaCount = 2;
+  private static int managedServersCount = 2;
+  private static String domain1Namespace = null;
+  private static String domain1Uid = "lboci-domain-1";
+  private static ConditionFactory withStandardRetryPolicy = null;
+
+  // constants for creating domain image using model in image
+
+  private static final String IMAGE_NAME = "ocilb-image";
+  private static final String SAMPLE_APP_NAME = "sample-app";
+  private static String cluster1Name = "cluster-1";
+
+  private static LoggingFacade logger = null;
+  private static String loadBalancerIP = null;
+  private static final String OCI_LB_NAME = "ocilb";
+
+  /**
+   * Install and verify operator.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(5, MINUTES).await();
+
+    logger.info("Get a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    final String opNamespace = namespaces.get(0);
+
+    logger.info("Get a unique namespace for WebLogic domain1");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domain1Namespace = namespaces.get(1);
+
+    logger.info("install and verify operator");
+    installAndVerifyOperator(opNamespace, domain1Namespace);
+  }
+
+  @AfterAll
+  public void tearDownAll() {
+    Kubernetes.deleteService(OCI_LB_NAME, domain1Namespace);
```

Comment:
You can add the check for "SKIP_CLEANUP" here:
if (System.getenv("SKIP_CLEANUP") == null
         || (System.getenv("SKIP_CLEANUP") != null
         && System.getenv("SKIP_CLEANUP").equalsIgnoreCase("false"))) {
         Kubernetes.deleteService(OCI_LB_NAME, domain1Namespace);
}

Reply:
done